site/layouts/partials/header.html

```diff
@@ -14,6 +14,7 @@
     </title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="{{"/css/style.css" | relURL}}" />
+    <link rel="icon" href="{{"/img/favicon.svg"}}" type="image/svg" />
```

Comment:
can you follow the `| relURL ` method as above. that basically ensures that if we someday host the site on /site, we can still have /site/img/favicon